src/Pages/HomePage/HomePage.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import "./HomePage.css"
+
+import Navbar from "../../Components/Navbar/Navbar"
+import Footer from "../../Components/Footer/Footer"
```

Comment:
same here too, create index.js file in components folder and export all components from it.

Reply:
`import Navbar from "../../Components/Navbar/Navbar"
import Footer from "../../Components/Footer/Footer"
import { DealsSection, BrandSection, ArrivalsSection, CategoriesSection, HeroSection } from './PageSections/index'`
is this alright? or do i have to put the navbar and footer in index.js too?